api/AnimationEvent.json

```diff
@@ -338,13 +338,13 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/AnimationEvent/pseudoElement",
           "support": {
             "webview_android": {
-              "version_added": false
+              "version_added": "68"
```

Comment:
It might be a good idea to move `webview_android` to the end of the `support` block (see issue #398 and PR #1882)

Reply:
I think the plan is to alphabetize everything once the PR queue has been gone-through, so you don't need to worry about that for now.